deploy/kubernetes/readme.md

```diff
@@ -36,10 +36,12 @@ Then push the image you tagged, by running the command:
 
 - Check that the image-names and images-tags specified in `altinncore-loadbalancer.yaml` and `altinncore-deployment.yaml` are correct with what you have in Azure ACR
 - Check that the images pull secret name is correct (To doublecheck, run the command `kubectl get sercrets`)
+- Make that a keyvault is created in the resource group, and a secret is created with the `altinn-appsettings-secret.json` file as data
```

Comment:
Please rewrite this sentence so it makes sense

Reply:
Better?